pkgs/applications/networking/cluster/ssm-agent/default.nix

```
@@ -96,8 +99,14 @@ buildGoPackage rec {
     popd
   '';
 
+  postInstall = ''
+    mkdir -p $out/etc
+    cp ${src}/amazon-ssm-agent.json.template $out/etc/.
+    cp ${src}/seelog_unix.xml $out/seelog.xml.template
```

Might as well mimic the upstream layout-ish.

```suggestion
    mkdir -p $out/etc/amazon/ssm
    cp amazon-ssm-agent.json.template $out/etc/amazon/ssm/amazon-ssm-agent.json.template
    cp seelog_unix.xml $out/etc/amazon/ssm/seelog.xml.template
```

---

When I remove `${src}/`, I get this:

```
cp: cannot stat 'amazon-ssm-agent.json.template': No such file or directory
```

I had to use the prefix `go/src/${goPackagePath}/` to find it.

---

Ah, right. I forgot about that.